src/index.js

```diff
@@ -1,23 +1,25 @@
 import changeCaseObject from 'change-case-object'
+import MetaOptions from './MetaOptions'
 import qs from 'query-string'
 import merge from 'deepmerge'
 
-const ALLOWED_REQUEST_METHODS = [
-  'GET',
-  'POST',
-  'PUT',
-  'DELETE',
-  'PATCH'
-]
+const ALLOWED_REQUEST_METHODS = ['GET', 'POST', 'PUT', 'DELETE', 'PATCH']
+
+const defaultMetaOptions = { bodyKeyCase: 'SNAKE_CASE' }
 
 export default class Snuffles {
-  constructor(baseUrl, defaultOptions = {}) {
+  constructor(
+    baseUrl,
+    defaultRequestOptions = {},
+    metaOptions = { ...defaultMetaOptions }
```

Comment:
If you don't modify `metaOptions` (which shouldn't be modified), you don't need to clone it.

```suggestion
    metaOptions = defaultMetaOptions
```